addons/l10n_hu/data/account.fiscal.position.tax.template.csv

```diff
@@ -3,21 +3,19 @@
 "fiscal_position_hu_exempt_tax_F18","F18","FA","fiscal_position_hu_exempt"
 "fiscal_position_hu_exempt_tax_F5","F5","FA","fiscal_position_hu_exempt"
 "fiscal_position_hu_exempt_tax_V27","V27","VA","fiscal_position_hu_exempt"
+"fiscal_position_hu_exempt_tax_V27LN","V27LN","VA","fiscal_position_hu_exempt"
+"fiscal_position_hu_exempt_tax_V27TE","V27TE","VA","fiscal_position_hu_exempt"
 "fiscal_position_hu_exempt_tax_V18","V18","VA","fiscal_position_hu_exempt"
+"fiscal_position_hu_exempt_tax_V18LN","V18LN","VA","fiscal_position_hu_exempt"
 "fiscal_position_hu_exempt_tax_V5","V5","VA","fiscal_position_hu_exempt"
-"fiscal_position_hu_eu_tax_F27","F27","FEU","fiscal_position_hu_eu"
-"fiscal_position_hu_eu_tax_F18","F18","FEU","fiscal_position_hu_eu"
-"fiscal_position_hu_eu_tax_F5","F5","FEU","fiscal_position_hu_eu"
-"fiscal_position_hu_eu_tax_FA","FA","FEU","fiscal_position_hu_eu"
-"fiscal_position_hu_eu_tax_V27","V27","VEU","fiscal_position_hu_eu"
-"fiscal_position_hu_eu_tax_V18","V18","VEU","fiscal_position_hu_eu"
-"fiscal_position_hu_eu_tax_V5","V5","VEU","fiscal_position_hu_eu"
-"fiscal_position_hu_eu_tax_VA","VA","VEU","fiscal_position_hu_eu"
-"fiscal_position_hu_eu_out_tax_F27","F27","FEUO","fiscal_position_hu_eu_out"
-"fiscal_position_hu_eu_out_tax_F18","F18","FEUO","fiscal_position_hu_eu_out"
-"fiscal_position_hu_eu_out_tax_F5","F5","FEUO","fiscal_position_hu_eu_out"
-"fiscal_position_hu_eu_out_tax_FA","FA","FEUO","fiscal_position_hu_eu_out"
-"fiscal_position_hu_eu_out_tax_V27","V27","VEUO","fiscal_position_hu_eu_out"
-"fiscal_position_hu_eu_out_tax_V18","V18","VEUO","fiscal_position_hu_eu_out"
-"fiscal_position_hu_eu_out_tax_V5","V5","VEUO","fiscal_position_hu_eu_out"
-"fiscal_position_hu_eu_out_tax_VA","VA","VEUO","fiscal_position_hu_eu_out"
+"fiscal_position_hu_exempt_tax_V5LN","V5LN","VA","fiscal_position_hu_exempt"
+"fiscal_position_hu_eu_F27","F27","FEUT","fiscal_position_hu_eu"
+"fiscal_position_hu_eu_F18","F18","FEUT","fiscal_position_hu_eu"
+"fiscal_position_hu_eu_F5","F5","FEUT","fiscal_position_hu_eu"
+"fiscal_position_hu_eu_V27","V27","VEU27T","fiscal_position_hu_eu"
+"fiscal_position_hu_eu_V27TE","V27TE","VEU27TE","fiscal_position_hu_eu"
+"fiscal_position_hu_eu_out_F27","F27","FEXT","fiscal_position_hu_eu_out"
+"fiscal_position_hu_eu_out_F18","F18","FEXT","fiscal_position_hu_eu_out"
+"fiscal_position_hu_eu_out_F5","F5","FEXT","fiscal_position_hu_eu_out"
+"fiscal_position_hu_eu_out_V27","V27","VIMK","fiscal_position_hu_eu_out"
+"fiscal_position_hu_eu_out_V27TE","V27TE","VIMK","fiscal_position_hu_eu_out"
```

Comment:
Can you configure your IDE to add a EOL at the end of every file (or every syntax used by Odoo)?